pkgs/games/wargus/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, callPackage, fetchFromGitHub
+, cmake, pkgconfig, makeWrapper
+, zlib, bzip2, libpng
+, dialog, python, cdparanoia
+}:
+
+let
+  stratagus = callPackage ./stratagus.nix {};
```

Comment:
I am not sure on this one. Maybe we can move it to libraries? Technically other packages could require it, right?

Reply:
I'm not sure either. Counter argument: many games reside with their engine packages in `pkgs/games/*/`